src/Chainweb/Mempool/InMem.hs

```diff
@@ -416,21 +468,20 @@ getPendingInMem cfg nonce lock since callback = do
 
   where
     sendAll psq = do
-        let keys = HashMap.keys psq
+        let keys = HM.keys psq
         (dl, sz) <- foldlM go initState keys
         void $ sendChunk dl sz
 
-    sendSome psq rlog (rNonce, oHw) = do
-        if rNonce /= nonce
-          then sendAll psq
-          else sendSince psq rlog oHw
+    sendSome psq rlog (rNonce, oHw)
+        | rNonce /= nonce = sendAll psq
```

Comment:
This is more unhelpful churn (re replacing the `if` with guards). Both sides are semantically identical, one doesn't read any better than the other, and use of `otherwise` causes false negatives in test coverage reports